.babelrc

```diff
@@ -1,5 +1,8 @@
 {
   "presets": [
+    ["@babel/preset-react", {
+      "pragma": "createElement"
```

Comment:
Is this pragma required here or is there a sensible default? (I was googling about pragma stuff for react and it seems like most react-related supportive tooling like plugins, whatnot, have a pragma default that is `createElement` or similar).

Reply:
It is required here I believe, see my comment about ESLint, I’ll double check though.